pkg/config/config.go

```diff
@@ -486,7 +487,8 @@ func NewConfig(userConfigPath string) (*Config, error) {
 		if err = readConfigFromFile(userConfigPath, config); err != nil {
 			return nil, errors.Wrapf(err, "reading user config %q", userConfigPath)
 		}
-		logrus.Debugf("Merged user config %q: %+v", userConfigPath, config)
+		logrus.Debugf("Merged user config %q", userConfigPath)
+		logrus.Tracef("%+v", config)
```

Comment:
Trace is not currently a supported log level.

https://github.com/containers/podman/blob/master/cmd/podman/common/completion.go#L24

Reply:
Ok we will need to add that support there as well, good catch.